### ISSUE TITLE
Connected Peers Flapping @ ETC 

### ISSUE THREAD

**Issue by MrMarioMichel** [hyperledger/besu#3463]
### Steps to Reproduce (Bug)
1. Use Version v22.1.0 in Docker
2. Use Ethereum Classic Chain
3. Full Sync

**Expected behavior:**

No "flapping" peers and try to connect to all found peers as peer limit is higher than found peers to connect to.

**Actual behavior:** 

At least 2 to 4 times a day. Even more often if you check the logs and not the data collected via the node exporter and represented in Grafana.

```
besu_1           | 2022-02-20 11:08:31.337+00:00 | main | INFO  | DefaultSynchronizer | Starting synchronizer.
besu_1           | 2022-02-20 11:08:31.361+00:00 | main | INFO  | FullSyncDownloader | Starting full sync.
besu_1           | 2022-02-20 11:08:31.364+00:00 | main | INFO  | FullSyncTargetManager | No sync target, waiting for peers. Current peers: 0
besu_1           | 2022-02-20 11:08:31.435+00:00 | main | INFO  | Runner | Ethereum main loop is up.
besu_1           | 2022-02-20 11:08:32.402+00:00 | nioEventLoopGroup-3-1 | INFO  | FullSyncTargetManager | No sync target, waiting for peers. Current peers: 1
besu_1           | 2022-02-20 11:08:37.406+00:00 | EthScheduler-Timer-0 | INFO  | FullSyncTargetManager | No sync target, waiting for peers. Current peers: 17
besu_1           | 2022-02-20 11:08:38.554+00:00 | nioEventLoopGroup-3-8 | INFO  | FullSyncTargetManager | No sync target, waiting for peers. Current peers: 66
besu_1           | 2022-02-20 11:08:40.069+00:00 | nioEventLoopGroup-3-2 | INFO  | FullSyncTargetManager | No sync target, waiting for peers. Current peers: 75
besu_1           | 2022-02-20 11:08:40.602+00:00 | nioEventLoopGroup-3-10 | INFO  | FullSyncTargetManager | No sync target, waiting for peers. Current peers: 67
besu_1           | 2022-02-20 11:08:40.703+00:00 | nioEventLoopGroup-3-7 | INFO  | FullSyncTargetManager | No sync target, waiting for peers. Current peers: 68
besu_1           | 2022-02-20 11:08:41.153+00:00 | nioEventLoopGroup-3-6 | INFO  | FullSyncTargetManager | No sync target, waiting for peers. Current peers: 70
besu_1           | 2022-02-20 11:08:41.315+00:00 | nioEventLoopGroup-3-6 | INFO  | FullSyncTargetManager | No sync target, waiting for peers. Current peers: 52
besu_1           | 2022-02-20 11:08:41.956+00:00 | nioEventLoopGroup-3-10 | INFO  | FullSyncTargetManager | No sync target, waiting for peers. Current peers: 37
besu_1           | 2022-02-20 11:08:43.120+00:00 | nioEventLoopGroup-3-3 | INFO  | FullSyncTargetManager | No sync target, waiting for peers. Current peers: 36
besu_1           | 2022-02-20 11:08:43.242+00:00 | nioEventLoopGroup-3-3 | INFO  | FullSyncTargetManager | No sync target, waiting for peers. Current peers: 35
besu_1           | 2022-02-20 11:08:43.678+00:00 | nioEventLoopGroup-3-2 | INFO  | FullSyncTargetManager | No sync target, waiting for peers. Current peers: 35
besu_1           | 2022-02-20 11:08:44.962+00:00 | nioEventLoopGroup-3-3 | INFO  | FullSyncTargetManager | No sync target, waiting for peers. Current peers: 36
besu_1           | 2022-02-20 11:08:45.811+00:00 | nioEventLoopGroup-3-10 | INFO  | FullSyncTargetManager | No sync target, waiting for peers. Current peers: 27
besu_1           | 2022-02-20 11:08:47.679+00:00 | nioEventLoopGroup-3-6 | INFO  | FullSyncTargetManager | No sync target, waiting for peers. Current peers: 23
besu_1           | 2022-02-20 11:08:48.714+00:00 | nioEventLoopGroup-3-10 | INFO  | BlockPropagationManager | Saving announced block 14575841 (0x92d816b994fd5ba1835c1b4849047dba8922d426c47291d4452f4f73e2ce0bf8) for future import
besu_1           | 2022-02-20 11:08:49.375+00:00 | nioEventLoopGroup-3-3 | INFO  | FullSyncTargetManager | No sync target, waiting for peers. Current peers: 44
besu_1           | 2022-02-20 11:08:50.143+00:00 | nioEventLoopGroup-3-2 | INFO  | FullSyncTargetManager | No sync target, waiting for peers. Current peers: 27
besu_1           | 2022-02-20 11:08:55.144+00:00 | EthScheduler-Timer-0 | INFO  | FullSyncTargetManager | No sync target, waiting for peers. Current peers: 14
besu_1           | 2022-02-20 11:09:00.148+00:00 | EthScheduler-Timer-0 | INFO  | FullSyncTargetManager | No sync target, waiting for peers. Current peers: 11
besu_1           | 2022-02-20 11:09:00.655+00:00 | nioEventLoopGroup-3-6 | INFO  | BlockPropagationManager | Saving announced block 14575842 (0x96b355aed97a10b8bcc0b43f58ade59266fb64f8811d26b8d72d64cafa4a1b06) for future import
besu_1           | 2022-02-20 11:09:01.550+00:00 | nioEventLoopGroup-3-7 | INFO  | FullSyncTargetManager | No sync target, waiting for peers. Current peers: 13
besu_1           | 2022-02-20 11:09:03.885+00:00 | nioEventLoopGroup-3-10 | INFO  | BlockPropagationManager | Saving announced block 14575843 (0x640278c5cc759ed1e483e3d55e26324dcb627e1a0378d8050678154d5fc3cd33) for future import
besu_1           | 2022-02-20 11:09:06.259+00:00 | nioEventLoopGroup-3-6 | INFO  | BlockPropagationManager | Saving announced block 14575844 (0x24311a8ece2c54500b906750d77892ed62905169f1359559f08628f75ebaa3f0) for future import
besu_1           | 2022-02-20 11:09:06.551+00:00 | EthScheduler-Timer-0 | INFO  | FullSyncTargetManager | No sync target, waiting for peers. Current peers: 10
besu_1           | 2022-02-20 11:09:10.103+00:00 | nioEventLoopGroup-3-8 | INFO  | SyncTargetManager | Found common ancestor with peer Peer 0xc25688b33f3e3eb997... at block 14575839
besu_1           | 2022-02-20 11:09:10.117+00:00 | nioEventLoopGroup-3-10 | INFO  | BlockPropagationManager | Saving announced block 14575845 (0xabfbaafe2a8b0877e1d831180d40a49c1c614555aa92f0f89a2db1fd5018355d) for future import
besu_1           | 2022-02-20 11:09:16.097+00:00 | EthScheduler-Workers-2 | INFO  | PersistBlockTask | Imported #14,575,841 / 0 tx / 0 om / 0 (0.0%) gas / (0x92d816b994fd5ba1835c1b4849047dba8922d426c47291d4452f4f73e2ce0bf8) in 0.027s. Peers: 10
besu_1           | 2022-02-20 11:09:16.101+00:00 | EthScheduler-Workers-2 | INFO  | BlockPropagationManager | Imported 1 pending blocks
besu_1           | 2022-02-20 11:09:16.342+00:00 | EthScheduler-Workers-0 | INFO  | PersistBlockTask | Imported #14,575,842 / 22 tx / 0 om / 501,074 (6.3%) gas / (0x96b355aed97a10b8bcc0b43f58ade59266fb64f8811d26b8d72d64cafa4a1b06) in 0.240s. Peers: 10
besu_1           | 2022-02-20 11:09:16.344+00:00 | EthScheduler-Workers-0 | INFO  | BlockPropagationManager | Imported 1 pending blocks
besu_1           | 2022-02-20 11:09:16.352+00:00 | EthScheduler-Workers-3 | INFO  | PersistBlockTask | Imported #14,575,843 / 0 tx / 0 om / 0 (0.0%) gas / (0x640278c5cc759ed1e483e3d55e26324dcb627e1a0378d8050678154d5fc3cd33) in 0.016s. Peers: 10
besu_1           | 2022-02-20 11:09:16.355+00:00 | EthScheduler-Workers-3 | INFO  | BlockPropagationManager | Imported 1 pending blocks
besu_1           | 2022-02-20 11:09:16.372+00:00 | EthScheduler-Workers-1 | INFO  | PersistBlockTask | Imported #14,575,844 / 1 tx / 0 om / 22,312 (0.3%) gas / (0x24311a8ece2c54500b906750d77892ed62905169f1359559f08628f75ebaa3f0) in 0.020s. Peers: 10
besu_1           | 2022-02-20 11:09:16.375+00:00 | EthScheduler-Workers-1 | INFO  | BlockPropagationManager | Imported 1 pending blocks
besu_1           | 2022-02-20 11:09:16.383+00:00 | EthScheduler-Workers-2 | INFO  | PersistBlockTask | Imported #14,575,845 / 0 tx / 0 om / 0 (0.0%) gas / (0xabfbaafe2a8b0877e1d831180d40a49c1c614555aa92f0f89a2db1fd5018355d) in 0.014s. Peers: 10
besu_1           | 2022-02-20 11:09:16.384+00:00 | EthScheduler-Workers-2 | INFO  | BlockPropagationManager | Imported 1 pending blocks
besu_1           | 2022-02-20 11:09:18.525+00:00 | EthScheduler-Workers-3 | INFO  | PersistBlockTask | Imported #14,575,846 / 0 tx / 0 om / 0 (0.0%) gas / (0x3de1fe9969e4c3b2c971299b6265d7bdee8f80b5b40f92e15c63a148ec617b3b) in 0.008s. Peers: 10
besu_1           | 2022-02-20 11:09:35.519+00:00 | EthScheduler-Workers-1 | INFO  | PersistBlockTask | Imported #14,575,847 / 4 tx / 0 om / 86,624 (1.1%) gas / (0x8adc4f85476cbb3cd6c2fe16552968664e8077f75356254a7c79dad8b7635372) in 0.030s. Peers: 10
--> besu_1           | 2022-02-20 11:09:47.799+00:00 | EthScheduler-Workers-2 | INFO  | PersistBlockTask | Imported #14,575,848 / 0 tx / 0 om / 0 (0.0%) gas / (0x8180634af7fea803bdbb3b27c7b5615dc2409f6d2e4d42ce9507547c3e9149d3) in 0.010s. Peers: 190
besu_1           | 2022-02-20 11:09:48.074+00:00 | nioEventLoopGroup-3-10 | INFO  | SyncTargetManager | Found common ancestor with peer Peer 0x3edfc67f36effaa7cd... at block 14575847
besu_1           | 2022-02-20 11:09:52.841+00:00 | EthScheduler-Workers-2 | INFO  | PersistBlockTask | Imported #14,575,849 / 0 tx / 0 om / 0 (0.0%) gas / (0x7e7ce09f1107f1d8072fa917320640733496fdcdc4d81c1c2baa03dd8bab41eb) in 0.021s. Peers: 16
besu_1           | 2022-02-20 11:10:12.009+00:00 | EthScheduler-Workers-0 | INFO  | PersistBlockTask | Imported #14,575,850 / 20 tx / 0 om / 423,936 (5.3%) gas / (0x233ee6492bb5f702490d5b93d4e80c1f9919438c749549cc62b52af2d248c724) in 0.053s. Peers: 15
besu_1           | 2022-02-20 11:10:13.443+00:00 | EthScheduler-Workers-1 | INFO  | PersistBlockTask | Imported #14,575,851 / 0 tx / 0 om / 0 (0.0%) gas / (0x7a11fadddf88cab7fab42efc0629086b135137a124d8160d22e2d3a238e5113e) in 0.005s. Peers: 15
besu_1           | 2022-02-20 11:10:36.953+00:00 | EthScheduler-Workers-1 | INFO  | PersistBlockTask | Imported #14,575,852 / 0 tx / 0 om / 0 (0.0%) gas / (0xa5f66608d737c314efc1025b5df76111136e5a191fa6d994726d70ade03cdc8a) in 0.007s. Peers: 14
besu_1           | 2022-02-20 11:10:38.082+00:00 | EthScheduler-Workers-2 | INFO  | PersistBlockTask | Imported #14,575,853 / 0 tx / 0 om / 0 (0.0%) gas / (0x6111a7b5511be55ef0497bd208946f1f042d62700c41165fedafcd641c578c6d) in 0.013s. Peers: 25
besu_1           | 2022-02-20 11:10:49.053+00:00 | EthScheduler-Workers-3 | INFO  | PersistBlockTask | Imported #14,575,854 / 0 tx / 0 om / 0 (0.0%) gas / (0x89b544c0175edc038ea2845069883dd6416066b6e484de9187b83b220358d87a) in 0.004s. Peers: 18
besu_1           | 2022-02-20 11:10:49.335+00:00 | EthScheduler-Workers-0 | INFO  | PersistBlockTask | Imported #14,575,855 / 6 tx / 0 om / 126,000 (1.6%) gas / (0x5bfab5a37324618528b99892a3d0f5bf77553f90d407de9e58f68a43cedc836f) in 0.033s. Peers: 17
besu_1           | 2022-02-20 11:11:02.455+00:00 | EthScheduler-Workers-2 | INFO  | PersistBlockTask | Imported #14,575,856 / 0 tx / 0 om / 0 (0.0%) gas / (0x27ac7df30a49f96ec55856996de13fecc00d221303c185ae8c2ca71c70d63db4) in 0.006s. Peers: 15
besu_1           | 2022-02-20 11:11:21.732+00:00 | EthScheduler-Workers-1 | INFO  | PersistBlockTask | Imported #14,575,857 / 8 tx / 0 om / 168,000 (2.1%) gas / (0x453f723e893b1ae2df6f909d3e7be60fbeb9b0e26ea64fe87299054b0ba6480f) in 0.029s. Peers: 14
besu_1           | 2022-02-20 11:11:33.865+00:00 | EthScheduler-Workers-3 | INFO  | PersistBlockTask | Imported #14,575,858 / 20 tx / 0 om / 420,000 (5.3%) gas / (0xc81239b3f559d04d34bb8e8a6142beedcce4b93a7aefcfc8f7ffee112832dab7) in 0.049s. Peers: 14
besu_1           | 2022-02-20 11:11:49.213+00:00 | EthScheduler-Workers-2 | INFO  | PersistBlockTask | Imported #14,575,859 / 30 tx / 0 om / 630,000 (7.9%) gas / (0xa2b0e6ae9ca06dfe410b952d1a1f5874a7a6d34f2f8b5914b99ebb668199b7f0) in 0.062s. Peers: 16
besu_1           | 2022-02-20 11:11:53.069+00:00 | EthScheduler-Workers-0 | INFO  | PersistBlockTask | Imported #14,575,860 / 6 tx / 0 om / 126,000 (1.6%) gas / (0xf0cefe4ed2b540707fa826048ecdd6f08cb9d7874a84331a6934d2abbf5938ac) in 0.016s. Peers: 15
besu_1           | 2022-02-20 11:12:07.450+00:00 | EthScheduler-Workers-0 | INFO  | PersistBlockTask | Imported #14,575,861 / 0 tx / 0 om / 0 (0.0%) gas / (0x1fda2aac8e36fd575682f77996055a9986139b3a2578de04c9fa346ed1ba1336) in 0.004s. Peers: 16
besu_1           | 2022-02-20 11:12:15.865+00:00 | EthScheduler-Workers-3 | INFO  | PersistBlockTask | Imported #14,575,862 / 49 tx / 0 om / 1,035,560 (12.9%) gas / (0x950f89703d11d910479a7d724dcd2cdfcf5dec278d9d314bbd65fd3f031703d8) in 0.097s. Peers: 14
besu_1           | 2022-02-20 11:12:52.152+00:00 | EthScheduler-Workers-2 | INFO  | PersistBlockTask | Imported #14,575,863 / 2 tx / 0 om / 42,000 (0.5%) gas / (0x04bec13d6ca2998b954f62f66f3c7a93c45be7b39c4ddd844a2688c3b171f8d2) in 0.013s. Peers: 19
besu_1           | 2022-02-20 11:13:11.455+00:00 | EthScheduler-Workers-3 | INFO  | PersistBlockTask | Imported #14,575,864 / 0 tx / 0 om / 0 (0.0%) gas / (0x635c233e7516c03b87b9fb30cc76818725893068e836694ffe11b40fd49afc34) in 0.005s. Peers: 16
besu_1           | 2022-02-20 11:13:11.767+00:00 | EthScheduler-Workers-2 | INFO  | PersistBlockTask | Imported #14,575,864 / 11 tx / 0 om / 234,936 (2.9%) gas / (0x2e094d7389ffb654d3f6bcab829e603cbfd25d322e14ae4496812a0be9239aa1) in 0.052s. Peers: 15
besu_1           | 2022-02-20 11:13:32.696+00:00 | EthScheduler-Workers-2 | INFO  | PersistBlockTask | Imported #14,575,865 / 0 tx / 1 om / 0 (0.0%) gas / (0xb32fc3883c9c2f18f755501fc72f4ecea9ea2a02b90a3a43d04f571ba6d2b2a2) in 0.015s. Peers: 14
--> besu_1           | 2022-02-20 11:13:42.314+00:00 | EthScheduler-Workers-3 | INFO  | PersistBlockTask | Imported #14,575,866 / 24 tx / 0 om / 514,336 (6.4%) gas / (0x79dde1d20f095ba1ffac298898da9819eab1c2bde773a4052a4ca1110892f51e) in 0.236s. Peers: 397
besu_1           | 2022-02-20 11:13:50.455+00:00 | EthScheduler-Workers-2 | INFO  | PersistBlockTask | Imported #14,575,867 / 1 tx / 0 om / 21,000 (0.3%) gas / (0x06a94a3c20e455aa75ffe259d1754145634d665e615890d1baf94e311755dd5e) in 0.009s. Peers: 15
besu_1           | 2022-02-20 11:14:05.689+00:00 | EthScheduler-Workers-1 | INFO  | PersistBlockTask | Imported #14,575,868 / 4 tx / 0 om / 467,399 (5.8%) gas / (0xab4cd210f24e9482ce1e6f9facce37d914f64540108852b6880e184fc318ec4e) in 0.135s. Peers: 15
besu_1           | 2022-02-20 11:14:10.828+00:00 | EthScheduler-Workers-0 | INFO  | PersistBlockTask | Imported #14,575,869 / 0 tx / 0 om / 0 (0.0%) gas / (0x3b9d7e2a1cd79f9e66ad913c5568a89613e8aeab7394582f868b21929c4a259c) in 0.005s. Peers: 15
besu_1           | 2022-02-20 11:14:13.305+00:00 | EthScheduler-Workers-3 | INFO  | PersistBlockTask | Imported #14,575,870 / 0 tx / 0 om / 0 (0.0%) gas / (0xb8d409d2859cc74d2d101825e08a8e0d03e717c81d50808fe0e1538dc8108fa0) in 0.006s. Peers: 14
besu_1           | 2022-02-20 11:14:48.000+00:00 | EthScheduler-Workers-2 | INFO  | PersistBlockTask | Imported #14,575,871 / 0 tx / 0 om / 0 (0.0%) gas / (0x367c432a9ab36d8815075b6a99f2476d29abab28ae4ecefe40f7c3f6b4a7cbc5) in 0.005s. Peers: 23
besu_1           | 2022-02-20 11:14:51.330+00:00 | EthScheduler-Workers-2 | INFO  | PersistBlockTask | Imported #14,575,872 / 39 tx / 0 om / 1,131,126 (14.1%) gas / (0x7db4825089705f87dd49bb0e477ee0fef6bd6093569517d6920789feb0086acd) in 0.186s. Peers: 19
besu_1           | 2022-02-20 11:15:18.820+00:00 | EthScheduler-Workers-3 | INFO  | PersistBlockTask | Imported #14,575,873 / 0 tx / 0 om / 0 (0.0%) gas / (0x15d475d75c301ec44ac5263b179f681c783a2066b6216c93b52fd23ed51bdc73) in 0.003s. Peers: 14
besu_1           | 2022-02-20 11:15:22.717+00:00 | EthScheduler-Workers-0 | INFO  | PersistBlockTask | Imported #14,575,874 / 0 tx / 0 om / 0 (0.0%) gas / (0xca8d6d717bdc50df2e8b18d7e5f2340e7f34afaa1170a6b89f1469e0563d7495) in 0.004s. Peers: 14
besu_1           | 2022-02-20 11:15:25.286+00:00 | EthScheduler-Workers-3 | INFO  | PersistBlockTask | Imported #14,575,875 / 6 tx / 0 om / 131,248 (1.6%) gas / (0x5447b8ca1c277610e407e42dabfdeb2d4fe23130b442f506112fc0972032dff3) in 0.019s. Peers: 14
besu_1           | 2022-02-20 11:15:34.099+00:00 | EthScheduler-Workers-1 | INFO  | PersistBlockTask | Imported #14,575,876 / 0 tx / 0 om / 0 (0.0%) gas / (0xf766fd41d610bfa967f1d177c8ce8686104881c4ed1b02fc25579447fb2bab5e) in 0.004s. Peers: 14
besu_1           | 2022-02-20 11:16:03.859+00:00 | EthScheduler-Workers-2 | INFO  | PersistBlockTask | Imported #14,575,877 / 9 tx / 0 om / 190,312 (2.4%) gas / (0x42f4d66c4d1885492e455472560ef39e5089cc9c6eef19ccf8300880698c98e6) in 0.030s. Peers: 18
besu_1           | 2022-02-20 11:16:09.731+00:00 | EthScheduler-Workers-1 | INFO  | PersistBlockTask | Imported #14,575,878 / 0 tx / 0 om / 0 (0.0%) gas / (0x0e18d3de1e17283c5d4fea29b596304ade836cbc4b66471fa9083547cbbb2650) in 0.006s. Peers: 16
besu_1           | 2022-02-20 11:16:21.978+00:00 | EthScheduler-Workers-3 | INFO  | PersistBlockTask | Imported #14,575,879 / 0 tx / 0 om / 0 (0.0%) gas / (0xef1c35353a99f1425480cb70a2f0c04768eda91e5ad8e3b31a2f72698ad23767) in 0.004s. Peers: 15
besu_1           | 2022-02-20 11:16:26.064+00:00 | EthScheduler-Workers-2 | INFO  | PersistBlockTask | Imported #14,575,880 / 1 tx / 0 om / 21,000 (0.3%) gas / (0xc7c8192027539b7d36550430ab44139228e404ba59ad501357cf8ef2957c3870) in 0.011s. Peers: 15
besu_1           | 2022-02-20 11:16:26.193+00:00 | nioEventLoopGroup-3-9 | INFO  | SyncTargetManager | Found common ancestor with peer Peer 0x9a28fae65fc6932951... at block 14575879
besu_1           | 2022-02-20 11:16:30.762+00:00 | EthScheduler-Workers-3 | INFO  | PersistBlockTask | Imported #14,575,881 / 17 tx / 0 om / 358,312 (4.5%) gas / (0x0e6cc0525704707acc54c3fda0ae12a00650febdc159d0b503136c337d9158a4) in 0.044s. Peers: 15
besu_1           | 2022-02-20 11:16:32.684+00:00 | EthScheduler-Workers-1 | INFO  | PersistBlockTask | Imported #14,575,882 / 0 tx / 0 om / 0 (0.0%) gas / (0x36d7d9dc7f03ece82fa3431ad277ec1beffa315daefb3a3ac274ad3b0fec9f2d) in 0.003s. Peers: 16
besu_1           | 2022-02-20 11:16:43.263+00:00 | EthScheduler-Workers-0 | INFO  | PersistBlockTask | Imported #14,575,883 / 24 tx / 0 om / 504,000 (6.3%) gas / (0x35f174d4138e112691df5d762257265d8d4357fa6c0f1a71b2f0a3d7a15afacd) in 0.085s. Peers: 22
besu_1           | 2022-02-20 11:16:53.654+00:00 | EthScheduler-Workers-1 | INFO  | PersistBlockTask | Imported #14,575,884 / 2 tx / 0 om / 42,000 (0.5%) gas / (0x89d4232e5b31b408f45c3a0d5cf2a8738cb7387a1b724dc458c2a14e3168d2fe) in 0.012s. Peers: 20
besu_1           | 2022-02-20 11:17:03.102+00:00 | EthScheduler-Workers-3 | INFO  | PersistBlockTask | Imported #14,575,885 / 0 tx / 0 om / 0 (0.0%) gas / (0x82f7a35c7f348c3f3ed0effaf0fe302fb30f93deea13c2414123741a7ad99cc3) in 0.003s. Peers: 14
besu_1           | 2022-02-20 11:17:09.698+00:00 | EthScheduler-Workers-3 | INFO  | PersistBlockTask | Imported #14,575,886 / 0 tx / 0 om / 0 (0.0%) gas / (0x706bbc20b84505675accf27acee9a36b6dd28f307666b580fd2c9f5281edd588) in 0.002s. Peers: 14
besu_1           | 2022-02-20 11:17:17.376+00:00 | EthScheduler-Workers-2 | INFO  | PersistBlockTask | Imported #14,575,887 / 15 tx / 0 om / 318,936 (4.0%) gas / (0x146e2545d5cd5f532905e478e0eecee0a27438e6b5bf767ea2bc15fc108d749b) in 0.032s. Peers: 13
besu_1           | 2022-02-20 11:17:22.773+00:00 | EthScheduler-Workers-1 | INFO  | PersistBlockTask | Imported #14,575,888 / 0 tx / 0 om / 0 (0.0%) gas / (0xdbfd5022bcc9026f1cadf8337d0ab2cafa7c2df08752f13c7fc0ee6edf8daf0c) in 0.004s. Peers: 14
besu_1           | 2022-02-20 11:17:29.755+00:00 | EthScheduler-Workers-1 | INFO  | PersistBlockTask | Imported #14,575,889 / 0 tx / 0 om / 0 (0.0%) gas / (0xf07ae46afce533b78749d0020f9405625ff817c3a531dc7fa1be7381d2c807b8) in 0.004s. Peers: 13
besu_1           | 2022-02-20 11:17:30.854+00:00 | EthScheduler-Workers-3 | INFO  | PersistBlockTask | Imported #14,575,890 / 0 tx / 0 om / 0 (0.0%) gas / (0x13bba3c0856effb96b6ec0bea22ded0f5b629a6953142d63b52b785166cfbd3d) in 0.003s. Peers: 13
besu_1           | 2022-02-20 11:17:43.141+00:00 | nioEventLoopGroup-3-4 | INFO  | BlockPropagationManager | Saving announced block 14575892 (0x4b319a92d8867a0f916949aafbccae625756808f39bbe8e257589a8bfdc1ae00) for future import
--> besu_1           | 2022-02-20 11:17:43.357+00:00 | EthScheduler-Workers-3 | INFO  | PersistBlockTask | Imported #14,575,891 / 28 tx / 0 om / 591,969 (7.4%) gas / (0x84a8b6e516e4f431aaf922960852a9b0ccf21c42f95518d0424ce732a253f11c) in 0.197s. Peers: 361
--> besu_1           | 2022-02-20 11:17:43.405+00:00 | EthScheduler-Workers-3 | INFO  | PersistBlockTask | Imported #14,575,892 / 1 tx / 0 om / 21,000 (0.3%) gas / (0x4b319a92d8867a0f916949aafbccae625756808f39bbe8e257589a8bfdc1ae00) in 0.047s. Peers: 355
besu_1           | 2022-02-20 11:17:43.406+00:00 | EthScheduler-Workers-3 | INFO  | BlockPropagationManager | Imported 1 pending blocks
besu_1           | 2022-02-20 11:17:48.364+00:00 | EthScheduler-Workers-0 | INFO  | PersistBlockTask | Imported #14,575,893 / 0 tx / 0 om / 0 (0.0%) gas / (0xe593352c0e8b25fb978b331fb675e0d70105690f58b6d4570af2bc834bd71ec8) in 0.003s. Peers: 15
besu_1           | 2022-02-20 11:18:04.239+00:00 | EthScheduler-Workers-3 | INFO  | PersistBlockTask | Imported #14,575,894 / 5 tx / 0 om / 106,312 (1.3%) gas / (0x6c65a16bf5614ab251d789cae007a2d1f26319dbb692747fc9006430204f4ddf) in 0.022s. Peers: 14
besu_1           | 2022-02-20 11:18:33.882+00:00 | EthScheduler-Workers-2 | INFO  | PersistBlockTask | Imported #14,575,895 / 9 tx / 0 om / 192,936 (2.4%) gas / (0xdf9c611e3b06ed77017982858ca6016629fe2a887c2406a51bce7d3ca64112bf) in 0.023s. Peers: 13
```
Versions (Add all that apply)

* Docker Version: Tag:v22.1.0

**Comment by macfarla** [comment]
dashboard: https://dashboard.etc-network.info/d/5S-6O8VZk/hyperledger-besu-node-at-ethereum-classic?orgId=1&var-Source=N%C3%BCrnberg%2FGermany%20(NUE)&from=now-24h&to=now&refresh=5s

**Comment by macfarla** [comment]
Spikes in the connected peers happens some times 1 - 2 times a day (increasing from 25ish to 50ish). And occasional spike increasing to 300 peers

**Comment by iamhsk** [comment]
This could be potentially resolved by #2689
